packages/replace/README.md

```diff
@@ -34,43 +34,25 @@ export default {
   input: 'src/index.js',
   output: {
     dir: 'output',
-    format: 'cjs'
+    format: 'cjs',
   },
   plugins: [
     replace({
-      __buildEnv__: 'production',
+      'process.env.NODE_ENV': JSON.stringify('production'),
       __buildDate__: () => new Date(),
```

Comment:
Should we json stringify this too?

Reply:
negative. it's important to show that it's possible. there's an older issue where this was added. I can't remember if it was the old repo or this one, but someone asked for this explicitly.

Reply:
strike that - I think the old ask was assuming that the `__buildDate__` would be in some sort of resource file, but that's not obvious at all. 
```suggestion
      __buildDate__: () => JSON.stringify(new Date()),
```